### PR TITLE
[Gtk] Add support for prelit

### PR DESCRIPTION
**Pull request by decriptor**

### CODE REVIEW

**Comment by slluis** [comment]
What is this needed for?

**Comment by sevoku** [comment]
This indicates whether a cell is currently highlighted (mouse over) or not. But it works only for Gtk this way, hence we'll need a more sophisticated solution. I think its time for a cell state as a param to the Draw event. I'll need to think about it next week.

File: Xwt.Gtk/Xwt.GtkBackend.CellViews/CustomCellRenderer.cs [hunk 1]
```diff
@@ -93,6 +95,14 @@ internal void EndDrawing ()
 				return base.HasFocus;
 			}
 		}
+
+		public override bool MouseOver {
+		 get {
```
Comment (sevoku) [inline Xwt.Gtk/Xwt.GtkBackend.CellViews/CustomCellRenderer.cs]: wrong formatting (spaces->tabs)

File: Xwt/Xwt/CellView.cs [hunk 2]
```diff
@@ -206,6 +206,10 @@ void ICellViewFrontend.DetachBackend ()
 			get { return Backend.HasFocus; }
 		}
 
+		protected bool MouseOver {
```
Comment (sevoku) [inline Xwt/Xwt/CellView.cs]: Wouldn't `IsHighlighted` be better?

**Pull request merged**